stable/aws-sigv4-proxy-admission-controller/templates/_helpers.tpl

```diff
@@ -0,0 +1,57 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "sidecar-injector.name" -}}
+{{- default .Chart.Name .Values.name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "sidecar-injector.fullname" -}}
+{{- printf "%s-%s" .Release.Name .Values.name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "sidecar-injector.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "sidecar-injector.labels" -}}
+helm.sh/chart: {{ include "sidecar-injector.chart" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Create the name of the service account to use
+*/}}
+{{- define "sidecar-injector.serviceAccountName" -}}
+{{- if .Values.serviceAccount.create }}
+{{- default (include "sidecar-injector.fullname" .) .Values.serviceAccount.name }}
+{{- else }}
+{{- default "default" .Values.serviceAccount.name }}
+{{- end }}
+{{- end }}
+
+{{/*
+Generate certificates for webhook
+*/}}
+{{- define "sidecar-injector.gen-certs" -}}
+{{- $serviceName := ( printf "%s-%s" ( include "sidecar-injector.fullname" . ) "webhook-svc" ) -}}
+{{- $cn := ( printf "%s.%s.%s" $serviceName .Release.Namespace "svc" ) -}}
+{{- $ca := genCA "kubernetes" 3650 -}}
+{{- $cert := genSignedCert $cn nil nil 3650 $ca -}}
```

Comment:
You need to specify the altNames not the CN. The trend is to ignore the CN during validation.